gen/builders/state_zero.go

```diff
@@ -50,8 +50,8 @@ const (
 	TestSealProofType = abi.RegisteredSealProof_StackedDrg2KiBV1
 )
 
-func (b *Builder) initializeZeroState(selector schema.Selector) {
-	if err := insertEmptyStructures(b.Stores.ADTStore); err != nil {
+func (st *StateTracker) initializeZeroState(selector schema.Selector) {
```

Comment:
move to state_tracker.go ?

Reply:
It's already logically grouped there. `state_zero.go` vs `state_tracker.go`.